js/script.js

```diff
@@ -0,0 +1,26 @@
+function readMore(click) { // <-- put event keyword here
+  click.preventDefault(); // <-- call event.preventDefault() first
```

Comment:
I'm not sure if you _have_ to use "event" when writing `event.preventDefault();`... 
If it worked, awesome, if not, stick to the suggested format.